.github/workflows/shared-run-e2e-tests.yaml

```diff
@@ -133,16 +130,37 @@ jobs:
       - name: Checkout uid2-shared-actions repo
         uses: actions/checkout@v3
         with:
-          ref: v2.7
+          ref: kcc-UID2-2339-create-test-pipeline-for-gcp
```

Comment:
I will update the branch name to v2 once https://github.com/IABTechLab/uid2-e2e/pull/6 is merged